class.jetpack.php

```diff
@@ -2834,6 +2837,22 @@ public static function get_additional_stat_data( $prefix = '' ) {
 		return $return;
 	}
 
+	private static function get_site_user_count() {
+		global $wpdb;
+
+		if ( function_exists( 'wp_is_large_network' ) ) {
+			if ( wp_is_large_network( 'users' ) ) {
+				return 10101; // Not a real value but should tell us that we are dealing with a large network.
```

Comment:
`wp_is_large_network()` requires a parameter that specifies whether to base the result on sites or users. The default is sites. See [source code](https://core.trac.wordpress.org/browser/tags/4.5.3/src/wp-includes/ms-functions.php#L2386).

If you pass in 'users', though, it just calls `get_user_count()` anyway, which means that potentially in this function we could end up counting users twice, but we will always call it at least once.